Directory.Packages.props

```diff
@@ -5,14 +5,14 @@
   <ItemGroup>
     <PackageVersion Include="Appium.WebDriver" Version="4.2.1" />
     <PackageVersion Include="CommunityToolkit.Mvvm" Version="8.2.0" />
-    <PackageVersion Include="CommunityToolkit.WinUI.Animations" Version="8.0.230907" />
-    <PackageVersion Include="CommunityToolkit.WinUI.Controls.Primitives" Version="8.0.230907" />
-    <PackageVersion Include="CommunityToolkit.WinUI.Controls.SettingsControls" Version="8.0.230907" />
-    <PackageVersion Include="CommunityToolkit.WinUI.Controls.Segmented" Version="8.0.230907" />
-    <PackageVersion Include="CommunityToolkit.WinUI.Controls.Sizers" Version="8.0.230907" />
+    <PackageVersion Include="CommunityToolkit.WinUI.Animations" Version="8.0.240109" />
+    <PackageVersion Include="CommunityToolkit.WinUI.Controls.Primitives" Version="8.0.240109" />
+    <PackageVersion Include="CommunityToolkit.WinUI.Controls.SettingsControls" Version="8.0.240109" />
+    <PackageVersion Include="CommunityToolkit.WinUI.Controls.Segmented" Version="8.0.240109" />
+    <PackageVersion Include="CommunityToolkit.WinUI.Controls.Sizers" Version="8.0.240109" />
     <PackageVersion Include="CommunityToolkit.WinUI.Collections " Version="8.0.230907" />
```

Comment:
Got it, the package version was duplicated because of this line with the extra space. 🤔